src/redux/wallet/reducer.js

```diff
@@ -21,6 +21,7 @@ const initialState = {
     isSubmitting: false
   },
   isFetching: false,
+  isFetched: false,
```

Comment:
wow we argued so much about name of variable for this purpose and missed this awesome option – _isFetched_
you should provide mass replace from _isReady_ to _isFetched_ then